numpy/lib/index_tricks.py

```diff
@@ -840,6 +840,21 @@ def fill_diagonal(a, val, wrap=False):
            [0, 4, 0, 0, 0],
            [0, 0, 4, 0, 0]])
 
+    The anti-diagonal can be filled by reversing the order of elements
+    using either `numpy.flipud` or `numpy.fliplr`.
+
+    >>> a = np.arange(9).reshape(3, 3);
+    >>> np.fill_diagonal(np.fliplr(a), 2)  # Horizontal flip
+    >>> a
+    array([[0, 1, 2],
+           [3, 2, 5],
+           [2, 7, 8]])
+    >>> np.fill_diagonal(np.flipud(a), 9)  # Vertical flip
```

Comment:
Same comment as before - your horizontal and vertical flip produce the same output. Can you use `np.fill_diagonal(np.flipud(a), [1, 2, 3])` or similar here instead, so that there's a difference in output?